lib/pbench/server/api/resources/query_apis/metadata_index/index_mappings.py

```diff
@@ -59,16 +59,22 @@ class IndexMappings(Resource):
     def __init__(self, logger: Logger):
         self.logger = logger
 
-    def get(self, index_name: AnyStr) -> Dict[AnyStr, Any]:
+    def get(self, index_key: AnyStr) -> Dict[AnyStr, Any]:
         """
-        Return mapping properties of the document specified by the index name in the URI.
-        For example, user can get the run document properties by making a GET request on
-        index/mappings/run. Similarly other index documents can be fetched by making a GET
-        request on appropriate index names.
-        We fetch the mapping by querying the template database. If the template is not found
-        in the database NOT_FOUND error will be raised.
+        Return mapping properties of the document specified by the index name
+        in the URI. For example, user can get the run document properties by
+        making a GET request on index/mappings/search. Similarly other index
+        documents can be fetched by making a GET request on appropriate index names.
```

Comment:
We should probably change "index name(s)" to something like "document selector keyword" (😝), since, on the outside, we really shouldn't be admitting that indices exist (that's an implementation detail).  What we're returning here is basically the list of fields in the document type, right?  And, that has "nothing" to do with the index (at least from the requestor's point of view).